src/BitTorrent-1.1.xml

```diff
@@ -502,15 +494,16 @@
          <p>The Notice below must appear in each file of the Source Code of any copy you distribute of the Licensed
          Product or any hereto. Contributors to any Modifications may add their own copyright notices to
          identify their own contributions.</p>
-         <p>License: 
-        <br/>The contents of this file are subject to the BitTorrent Open Source License Version 1.0 (the
+         <p>License:</p>
+         <standardLicenseHeader>
+           <p>The contents of this file are subject to the BitTorrent Open Source License Version <alt name="version" match="1\.[01]">1.1</alt> (the
```

Comment:
Instead of rolling forward with the typo'd “1.0”, I've changed our canonical version to “1.1” (while continuing to allow the typo match).

Reply:
> Another request for moving this to a separate PR since so I can merge in the technical only changes.

Where do we draw the line for technical changes?  Does the [`>` typo fix][1] count?  Does the [variable GFDL markup][2] count?  I'm not clear on this particular instance being any different, and think it would be easiest to just get legal to review the changes (which I've listed out in 4dcd50b1f8).  If you'd rather me file those fixups *before* this PR, I can do that, although in that case we'll have to update/review both locations.

[1]: https://github.com/spdx/license-list-XML/pull/581#discussion_r161873752
[2]: https://github.com/spdx/license-list-XML/pull/581#discussion_r161873308

Reply:
>  If you'd rather me file those fixups before this PR, I can do that, although in that case we'll have to update/review both locations.

Nevermind.  I can just drop `BitTorrent-1.1` from this PR, which I've done with 4dcd50b → c526576.  I've filed #603 on top of this PR with the `BitTorrent-1.1` change.

Reply:
@wking Thanks for splitting out the separate PR.  I considered flagging the typos as well, but decided that it was OK since the text body didn't change.  In general, I was just trying to review structural changes and any necessary changes to the alt/optional text to allow matching both for the headerText and license text.